code/game/objects/effects/decals/crayon.dm

```diff
@@ -47,3 +47,289 @@ GLOBAL_LIST(gang_tags)
 /obj/effect/decal/cleanable/crayon/gang/Destroy()
 	LAZYREMOVE(GLOB.gang_tags, src)
 	..()
+
+// Mapping
+
+/obj/effect/decal/cleanable/crayon/rune
+	name = "Rune"
+	icon_state = "rune1"
+
+/obj/effect/decal/cleanable/crayon/rune/rune1
```

Comment:
also these (the "1" paths) are all redundant, its 100% identical to the base path and changes nothing, as such its a duplicate item shown in sdmms list when its not needed